main.cpp

```diff
@@ -35,89 +35,109 @@ arm.rotateCounterClockwise(); //demonstrates CamelCase
 
 /*
  1)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: missile
+ action 1: the missile is fired
+ action 2: the missile locks onto a target
+ action 3: the missile explodes
  */
-
+missile.fire();
+missile.selectTarget();
+missile.explode
 
 /*
  2)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: apple
+ action 1: the apple is eaten
+ action 2: the apple is sliced
+ action 3: the apple is thrown
  */
-
+apple.eat();
+apple.slice();
+apple.toss();  //looks like throw is already a defined function
 /*
  3)
- Noun:
- action 1:
- action 2:
- action 3:
+ Noun: laptop
+ action 1: the laptop is powered on
+ action 2: the laptops fans turn on
+ action 3: the laptops brightness turns down
  */
-
+laptop.powerOn();
+laptop.fanPowerOn();
+laptop.brightnessDown();
```

Comment:
Statements like `is powered on` come across in code as "getters".  meaning, you're trying to get the state of something.  In this case, that state is whether or not the laptop is powered on.  The pseudo code you wrote `laptop.powerOn()` is different from the original English action, and thus it is not truly representing what you intended.   your action 1 should be "the laptop powers on".   If English is your first language, these sentences could be described as 3rd person, present tense.    

- [ ] try to get your 3 actions to not involve the word 'is'.